bundles/org.openhab.binding.luftdateninfo/src/main/java/org/openhab/binding/luftdateninfo/internal/dto/Location.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.luftdateninfo.internal.dto;
+
+/**
+ * The {@link LuftdatenInfo} class definition for Logging identification
+ *
+ * @author Bernd Weymann - Initial contribution
+ */
+public class Location {
+    private Integer id;
+    private String country;
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    public String getCountry() {
+        return country;
+    }
+
+    public void setCountry(String country) {
+        this.country = country;
+    }
+
+    public String getAltitude() {
+        return altitude;
+    }
+
+    public void setAltitude(String altitude) {
+        this.altitude = altitude;
+    }
+
+    public String getLatitude() {
+        return latitude;
+    }
+
+    public void setLatitude(String latitude) {
+        this.latitude = latitude;
+    }
+
+    public String getLongtitude() {
+        return longtitude;
+    }
+
+    public void setLongtitude(String longtitude) {
+        this.longtitude = longtitude;
+    }
+
+    public Integer getIndoor() {
+        return indoor;
+    }
+
+    public void setIndoor(Integer indoor) {
+        this.indoor = indoor;
+    }
+
+    public Integer getExact_location() {
+        return exact_location;
+    }
+
+    public void setExact_location(Integer exact_location) {
+        this.exact_location = exact_location;
+    }
+
+    private String altitude;
+    private String latitude;
+    private String longtitude;
+    private Integer indoor;
+    private Integer exact_location;
```

Comment:
Please use camel case.
```suggestion
    private String altitude;
    private String latitude;
    private String longtitude;
    private Integer indoor;
    private Integer exactLocation;
```

Reply:
I think I cannot change this - name has to match exactly to the deliverd json key, right? That's why I ignored all checkstyle warnings in dto subfolder regarding camel case.

Reply:
You can annotate the field with `@SerializedName("exact_location")`. The field name can be anything, then.

Reply:
Perfect - looks good now and all checkstyle warnings are eliminated!